src/Logging/InitializationLogger.cs

```diff
@@ -24,7 +24,11 @@ public static class InitializationLogger
 
 		private static ILoggingBuilder LoadInitializationLogger(this ILoggingBuilder builder)
 		{
-			builder.AddConsole();
+			if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development")
```

Comment:
>Development [](http://example.com/codeflow?start=72&length=11)

do we have data if anyone is actually using that in Dev environment?

Reply:
I would be in favor in removing it in all environments and re-adding it if someone screams

Reply:
No I don't have that data, do you know how we would get that? Wouldn't it be safer to just keep it in dev and it shouldn't cause any problems/issues like it does in other environments?

Reply:
I think the more unused functionality we have, the more difficult it is to maintain it. Also - what if people see some logs in Dev and expect to see them in PROD? You have to explain about the console logger not being enabled.

I would just add OMEX to this PR.

Reply:
The console logger would be where people see logs during development? I think it should be OK to keep it in such circumstances, I don't think many people would be logging onto the fixed environments and expect to see the console for their 1 app running.

Reply:
I am checking the Diagnostic events during development, it would be good to check locally what happens when this logger is disabled - it might be the case that the console logger just adds some logging before initializing the other loggers.